### PR TITLE
Fix issue in tree

### PR DESCRIPTION
**Pull request by Daniel-Roberts-Bose**
Use stable environment and not master

### CODE REVIEW

**Comment by Michael-Cook-Bose** [comment]
"Fix issue in tree"? That's the best we can do for a description of this change? :-)

**Comment by Daniel-Roberts-Bose** [comment]
Well I am still trying to root cause the problem but the actual error that is being thrown is
```
JSON file created at '/tmp/tmp.LjxucHT5i3/create.json'
ERROR: opensource-nghttp2/1.24.1-28+g1990fa1@BoseCorp/bose-master: Incompatible requirements obtained in different evaluations of 'requirements'
    Previous requirements: [opensource-zlib/1.2.12-21+ge1736ca@BoseCorp/bose-master]
    New requirements: [opensource-zlib/1.2.12-21+ge1736ca@BoseCorp/conan_stable]
```
But whats confusing is that it should of just rebuilt nghttp based on the `conan_stable` build

File: Bose/depends.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 [
   {
-    "channel": "bose-master",
+    "channel": "conan_stable",
     "component": "opensource-zlib",
     "options": "",
     "user": "BoseCorp",
-    "version": "[>=1.2.11-17, include_prerelease=True]"
+    "version": "[>=0.0.1-0, include_prerelease=True]"
```
Comment (Josh-Grauman-Bose) [inline Bose/depends.json]: Do you actually need to change the version number, or is it possible to get by with just changing the channel?

Comment (Daniel-Roberts-Bose) [reply Bose/depends.json]: I think its safer to, since the metabuild is using this

**Comment by Daniel-Roberts-Bose** [comment]
@Michael-Cook-Bose Do you think this PR is ok to merge?

**Pull request merged**